src/buildercore/terraform.py

```diff
@@ -580,13 +809,16 @@ class TerraformTemplateError(RuntimeError):
     pass
 
 class TerraformTemplate():
-    def __init__(self, resource=None, data=None):
+    def __init__(self, resource=None, data=None, locals=None):
```

Comment:
shadowing. I don't particular mind but it's weird the lint script didn't pick it up.

Reply:
It was a strange color in Vim.